deploy/kubernetes/examples/demo-storageclass.yaml

```diff
@@ -0,0 +1,16 @@
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: demo-storageclass
+provisioner: block.csi.ibm.com
+parameters:
+  SpaceEfficiency: deduplicated   # Optional.
+  pool: demo-pool
```

Comment:
@ArbelNathan do you think we should mention that this is only mandatory for ds8k or something?